chelsea.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
```

Comment:
Do we really need a `chelsea.cmd`? I'd been Googling around to try and see exactly what you need for a cross platform Ruby CLI app (surprisingly hard to find).

Reply:
I don't think it's strictly _needed_ but it's a convenience feature to save some keystrokes plus people may not be sure how to go about launching the CLI . Usually with DevAudit and the ther tools we'd have a .cmd and .sh launcher scripts for Windows and Linux.